add iot topic (#2945)
github/explore: topics/iot/index.md

> --

Reviewer: ```suggestion
---
```

> aliases: IoT, Internet-of-things

Reviewer 2: These are case insensitive so we don’t want a duplicate of the topic name here. 
```suggestion
aliases: Internet-of-things
```

display_name: Internet of things
related: cloud, embedded, devices

> short_description: The iot is a network of devices which exchanging data with other devices and systems over the internet.

Reviewer 2: ```suggestion
short_description: The iot is a network of devices that exchange data with other devices and systems over the internet.
```
